### PR TITLE
Updating terminology Community Connector > Marketplace Connector, Certified Connector > Integration

### PR DESCRIPTION
**Pull request by Adorism**
update nomenclature for connector types and spotted some small typos along the way

## Can this PR be safely reverted and rolled back?

- [X] YES 💚
- [ ] NO ❌

### CODE REVIEW

**Comment by Adorism** [comment]
Next step would be to update the connector in the registry, I think, which should also update the tables in the catalog.

File: docs/contributing-to-airbyte/README.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ Before getting started, please review Airbyte's Code of Conduct. Everyone intera
 ## Code Contributions
 
 Most of the issues that are open for contributions are tagged with [`good-first-issue`](https://github.com/airbytehq/airbyte/issues?q=is%3Aopen+is%3Aissue+label%3Agood-first-issue) or [`help-welcome`](https://github.com/airbytehq/airbyte/labels/help-welcome).
-A great place to start looking will be our GitHub projects for [**Community Connector Issues**](https://github.com/orgs/airbytehq/projects/50)
+A great place to start looking will be our GitHub projects for [**Marketplace Connector Issues**](https://github.com/orgs/airbytehq/projects/50)
```
Comment (natikgadzhi) [inline docs/contributing-to-airbyte/README.md:16]:
```suggestion
```

This link is to a project where we have _some_ but not all relevant issues, and a LOT of issues are in the closed repository. Until @marcosmarxm and I fix it, let's remove this link. I feel pretty strongly about this.

File: docs/contributing-to-airbyte/README.md [hunk 2]
```diff
@@ -13,9 +13,9 @@ Before getting started, please review Airbyte's Code of Conduct. Everyone intera
...
 
-Contributions for new connectors should come with documentation. Refer to the [guidelines on updateing documentation](writing-docs.md) which include a template to use while documentiong a new connector. 
+Contributions for new connectors should come with documentation. Refer to the [guidelines on updating documentation](writing-docs.md) which includes a template to use while documenting a new connector.
```
Comment (natikgadzhi) [inline docs/contributing-to-airbyte/README.md:18]:
```suggestion
Please include documentation when contributing a new connector. Refer to the [guidelines on updating documentation](writing-docs.md) which includes a template to use while documenting a new connector.
```

@marcosmarxm, one thing that changes with Builder Contribution flow is that we are going to auto-generate documentation for connectors based on their manifest, so separate docs are no longer going to be a thing. But for now, this stands.

File: docs/integrations/connector-support-levels.md [hunk 3]
```diff
@@ -6,54 +6,54 @@ products: all
...
+- Integrations are official Airbyte connectors that are available to all users.
+- These connectors have been tested and vetted. They are production ready.
+- Integrations should go through minimal breaking change but in the event an upgrade is
   needed users will be given an adequate upgrade window.
 
-## Community
+## Marketplace
 
-A **Community** connector is maintained by the Airbyte community until it becomes Certified. Airbyte
+A **Marketplace** connector is maintained by the Airbyte community until it becomes an Integration. Airbyte
```
Comment (natikgadzhi) [inline docs/integrations/connector-support-levels.md:42]:
```suggestion
A **Marketplace** connector is maintained by the community members until it becomes an official Integration. Airbyte
```

Comment (natikgadzhi) [reply docs/integrations/connector-support-levels.md:42]: To put more emphasis on community and less on Airbyte. Set the right support level expectation.

File: docs/integrations/connector-support-levels.md [hunk 4]
```diff
@@ -6,54 +6,54 @@ products: all
...
-- Community connectors may be upgraded to Certified at any time, and we will notify users of these
+- Marketplace connectors are available to all users.
+- Marketplace connectors may be upgraded to an official Integration at any time, and we will notify users of these
   upgrades via our Slack Community and in our Connector Catalog.
-- Community connectors might not be feature-complete (features planned for release are under
+- Marketplace connectors might not be feature-complete (features planned for release are under
   development or not prioritized) and may include backward-incompatible/breaking API changes with no
   or short notice.
-- Community connectors have no Support SLAs.
+- Marketplace connectors have no Support SLAs.
```
Comment (natikgadzhi) [inline docs/integrations/connector-support-levels.md:56]:
```suggestion
- Marketplace connectors have no Support SLAs.
- You're very welcome to contribute new features and streams to an existing Marketplace integration. Airbyte Contributor Experience team is happy to review PRs when we have capacity.
```

File: docs/understanding-airbyte/database-data-catalog.md [hunk 5]
```diff
@@ -8,7 +8,7 @@
   - Each record represents a connector that Airbyte supports, e.g. Postgres. This table represents all the connectors that is supported by the current running platform.
   - The `actor_type` column tells us whether the record represents a Source or a Destination.
   - The `spec` column is a JSON blob. The schema of this JSON blob matches the [spec](airbyte-protocol.md#actor-specification) model in the Airbyte Protocol. Because the protocol object is JSON, this has to be a JSON blob.
-  - The `support_level` describes the support level of the connector (e.g. community, certified).
+  - The `support_level` describes the support level of the connector (e.g. Marketplace, Integration).
```
Comment (natikgadzhi) [inline docs/understanding-airbyte/database-data-catalog.md:11]:
```suggestion
  - The `support_level` describes the support level of the connector (e.g. `community`, `certified`, or `archived`).
    - In the product UI, Marketplace contains connectors with `community` support level, and Integrations tab contains `certified` connectors.
    - `support_level: archived` signals that this connector is no longer supported, and it's not available to install for any new connections. 
```

**Review by natikgadzhi — APPROVED** [review]
Overall ok, approving to unblock. Please review my suggestions and accept them if the language is ok. I feel pretty strongly about adding them in /cc @Adorism

**Pull request merged**